.github/workflows/release.yml

```
@@ -16,19 +22,41 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-      - name: Build
+      - name: Build Nuxt
         run: npm run build
 
+      - name: Generate static site
+        run: npm run generate
+
       - name: Create Release
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.11)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)